platforms/minigame/platforms/taobao/wrapper/builtin/Image.js

```diff
@@ -0,0 +1,40 @@
+// var screencanvas = $global.screencanvas;
+
+function Image () {
+    // empty constructor
+}
+
+// NOTE: Proxy not supported on iOS 8 and 9.
+// let ImageProxy = new Proxy(Image, {
+//     construct (target, args) {
+//         let img =  screencanvas.createImage();
+//         if (!img.addEventListener) {
```

Comment:
Image can not work now?

Reply:
The image can work now . The previous code has not been removed, it has just been commented out.

Reply:
If so, then please remove it and add comment.

Reply:
OK.